src/controllers/message.js

```diff
@@ -17,4 +18,28 @@ const createMessage = async (req, res) => {
     res.status(StatusCodes.CREATED).json({ message })
 }
 
-module.exports = { createMessage }
+// Get message conversation
+const getMessageConversation = async (req, res) => {
+    const sort = { createdAt: 1 }
```

Comment:
Lara I think, you can simply assign the value of sort into sort itself after find, but the point is that it should be (-1), which will return the most recent value!
.sort({ createdAt: -1 })

Reply:
Thank you for the review Manizha! Well noted on sort, but on my ticket, it was written that I need to return the recent message at the end. @jpremo could you please advise on this?

Reply:
@larasodjati  you are correct. We want it sorted so that the most recent message is at the end of the array.

Reply:
@jpremo Thanks for the confirmation, could you please check and approve this PR?